WTWSMS/events/WtWSMS_zusk_misc_events.txt

```diff
@@ -571,12 +568,17 @@ character_event = {
 				culture_group = central_germanic
 		        culture_group = east_germanic
 		        culture_group = west_germanic
+				culture_group = byzantine
```

Comment:
The ERE is the Roman Empire, but it's not the Roman Empire in the "good old days", when it stretched across the Medditerranean Sea. That's why they still get nostalgia.